### PR TITLE
Copy-As-ToDo (Fixes #9302)

### LINKED ISSUE #9302: add "Copy As To-Do" to Tavern, guilds, party
The "Copy As To-Do" feature needs to be added back to **all** messages in the Tavern, all guilds, and the user's party. 

`website/client/components/chat/chatMessages.vue` is where you'd start looking into this; you'll see that there's already code there for it but it's been commented out because the copyAsTodo button wasn't working. The contributor who works on this would need to work out why and fix the problem. The feature was working on the old website so it's possible that the site's JavaScript code and/or API have support for it.

We do also want this feature to work for private messages in the Inbox, however that might need to be fixed in a different / more complex way. I'm not sure if that ever worked on the old website. If the contributor who works on this issue would like to fix it in the Inbox too, that would be great, but it's also okay if you just get it working in the Tavern, guilds, and party for now. We can open a new issue for the inbox later. If you don't get it working in the inbox, make sure that the copyAsTodo icon doesn't appear there.

### PR DESCRIPTION
**Pull request by perkyguy**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #9302

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
- Enabled the Copy To Task element for chat messages
- The Copy To Task component will now take in a copied message and create a task, and a notification
- The `group.vue` and `tavern.vue` files weren't binding the `group-name`, and so I fixed those real quick as well.

This **does not** add the functionality to the inbox. The change to have it work from there is trivial, but getting the modals to behave funny is another story. I will post my findings in the issue itself. 

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID: e8be8049-ac14-401f-89ff-ce0aaa3445b4

### CODE REVIEW

File: website/client/components/chat/copyAsTodoModal.vue [hunk 1]
```diff
@@ -50,18 +55,22 @@ export default {
...
 
-      //  @TODO: Add after tasks: User.addTask({body:newTask});
-      // @TODO: Notification.text(window.env.t('messageAddedAsToDo'));
+      this.createTask(newTask);
+      this.$store.dispatch('snackbars:add', {
+        title: 'Copy Task',
+        text: this.$t('messageAddedAsToDo'),
+        type: 'success',
+        timeout: true,
+      });
```
Comment (paglias) [inline website/client/components/chat/copyAsTodoModal.vue:73]: sorry for the delay reviewing this, the code looks good but can you make sure the `text` and `notes` in the modal get reset after the task is created? Thanks

**Review by paglias — CHANGES_REQUESTED** [review]

**Comment by paglias** [comment]
hey @perkyguy how are you doing on this?

**Comment by perkyguy** [comment]
Sorry, had a bunch of things going on, hadn't gotten back to this. I'll take care of this soon. I know I was just having some issues getting my dev environment to work correctly (all of the menus were broken). I'll get to this (and my other PRs) fixed and resubmitted soon!

**Comment by paglias** [comment]
I'm closing this since it seems to have been fixed elsewhere

**Pull request closed without merging**